aframe/src/index.js

```diff
@@ -63,16 +65,21 @@ async function createBanner(el, adUnit, format, style, height, beacon) {
   plane.setAttribute('class', 'clickable'); // required for BE
   el.appendChild(plane);
 
-  const bannerPromise = loadBanner(adUnit, format, style, beacon).then(banner => {
-    if (banner.img) {
-      assets.appendChild(banner.img);
-    }
-    return banner;
-  });
-
-  //setInterval(() => {
+  const getBanner = () => {
+    const bannerPromise = loadBanner(adUnit, format, style, beacon).then(banner => {
+      if (banner.img) {
+        assets.appendChild(banner.img);
+      }
+      return banner;
+    });
+  
     bannerPromise.then(banner => updateBanner(banner, plane, el, adUnit, format, style, height, beacon));
-  //}, 5000);
+  }
+
+  getBanner();
+  if (isBeta) {
+    setInterval(getBanner, 30000);
```

Comment:
Would be helpful to have this number be a descriptive constant so we know what the 30000 means

Reply:
Done